examples/azure/packages/.yarnrc.yml

```diff
@@ -1 +1 @@
-nodeLinker: node-modules
+nodeLinker: node - modules;
```

Comment:
This looks wrong to me. Apparently, prettier treaded the line as expression (like "a minus b") and therefore added spacing for readability, but actually, YAML has [no arithmetic expressions](https://yaml.org/spec/1.2.2/), and it should be interpreted [as string](https://yarnpkg.com/configuration/yarnrc#nodeLinker) like

```yaml
nodeLinker: "node-modules"
```

A quick fix could be adding those quotes, but I think it makes more sense to find out why prettier does handle yaml files like js or if it's actually treated as yaml and the formatter is just erroneous cause of the missing quotes.

Reply:
Ah and now that the line is updated: YAML does not have semicolons on line endings.

Reply:
There was the parser option set to TypeScript in the .prettierrc. According to the issue linked below prettier does not need this option and it will rather mess with its configuration if set in the top level. I removed the option and now it seems to work. Any idea if this why this was set in the first place? 
https://github.com/actions/typescript-action/issues/47

Reply:
Good catch, no, not sure why this was configured, probably cause we didn't know better. :)